test_files/molecules/new_qchem_files/almo.out

```diff
@@ -0,0 +1,485 @@
+
```

Comment:
Just a nitpick but maybe we want to rename this test files dir? From my experience `new_<sth>` names don't age that well.

Reply:
I'm not opposed to such a change (I personally don't care what the directory name is at all). But is this relevant to the present PR? That name has been in place for years.

Reply:
Oh, didn't realize. Not too familiar with the QChem side of pymatgen. But sort of makes my point that the name is confusing. `new` is meaningless after a while. A date or a QChem version number would be better. Feel free to ignore though since not specific to this PR. Except maybe now is as good a point as any to fix.

PR looks good! So if no other comments appear, happy to merge.